status-im/status-desktop
# status-go: share image/username on separate topic on a community

**test-only commit**

status-go PR: https://github.com/status-im/status-go/pull/2745

see: https://github.com/status-im/status-go/pull/2745#issuecomment-1187219757

### Summary

Now when user joins a community it will notify other members about their profile picture, ens name, etc.

### Testing notes
ens name propagation doesn't work, but this is global issue, not related to this PR

##### Functional

- communities

### Steps to test


- create test community
- create new userA
- join with user A to community
- create new userB
- join with userB to community
- userB adds/modifies profile picture
- userA should see updated userB profile picture

_(it's important to create new users to ensure they haven't met each other on any public chat before)_

https://user-images.githubusercontent.com/33099791/179510046-4c703ace-e434-43c9-a77f-e25482a20b47.mp4

https://user-images.githubusercontent.com/33099791/179510062-712bfab2-e7bb-468a-90da-1b4e74973c2f.mp4
